src/services/fetchers/GoldApiPriceFetcher.ts

```diff
@@ -28,10 +28,13 @@ export default class GoldApiPriceFetcher {
     this.timeout = settings.api.goldApi.timeout;
   }
 
-  async apply(input: GoldApiInputParams): Promise<number> {
-    this.logger.debug(`[GoldApiPriceFetcher] call for: ${input.symbol}/${input.currency}`);
+  async apply(params: GoldApiInputParams, options: FeedFetcherOptions): Promise<number> {
+    const {symbol, currency} = params;
+    const {base: feedBase, quote: feedQuote, timestamp} = options;
```

Comment:
```suggestion
    const {base: feedBase, quote: feedQuote} = options;
```

please remove timestamp in places where it is not in use